frontend/src/constant/RestConstants.ts

```diff
@@ -0,0 +1,4 @@
+const port = 8080;
+const server = `http://localhost:${port}`;
```

Comment:
kitam pr'e šitą vietą (http://localhost:8080) iškelk į .env failą
https://create-react-app.dev/docs/adding-custom-environment-variables/